### PR TITLE
adding my new project-create ros-speech-command-node.yml

### PR DESCRIPTION
**Pull request by Abdallah-Hesham99**
I would like to add to up-for-grabs !
kindly check my project, thanks a lot : )

### CODE REVIEW

**Review by Abdallah-Hesham99 — COMMENTED** [review]
fixed the issue

**Review by Abdallah-Hesham99 — COMMENTED** [review]
updated description

**Review by Abdallah-Hesham99 — COMMENTED** [review]
updated the label

**Review by Abdallah-Hesham99 — COMMENTED** [review]
updated formating

**Review by Abdallah-Hesham99 — COMMENTED** [review]
added underscores

**Review by Abdallah-Hesham99 — COMMENTED** [review]
updated formatting

File: _data/projects/ros-speech-command-node.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+name: tell_your_motor_speed
+desc: A ROS1 node that uses a TensorFlow model to recognize speech commands and then send a signal to Arduino which uses PID control to apply a corresponding speed to a DC motor
+site: https://github.com/Abdallah-Hesham99/tell_your_motor_speed
+tags:
+- pid_control
+- ros_noetic
+- tensorflow
+- deep_learning
+upforgrabs:
+  name: update to ros2 and micro-ros
```
Comment (shiftkey) [inline _data/projects/ros-speech-command-node.yml]:
```suggestion
  name: up for grabs
```

File: _data/projects/ros-speech-command-node.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+--- 
+name: tell_your_motor_speed
+desc: A ROS1 node that uses a TensorFlow model to recognize speech commands and then send a signal to Arduino which uses PID control to apply a corresponding speed to a DC motor
+site: https://github.com/Abdallah-Hesham99/tell_your_motor_speed
+tags:
+- pid_control
+- ros_noetic
+- tensorflow
+- deep_learning
```
Comment (shiftkey) [inline _data/projects/ros-speech-command-node.yml:9]: To make these tags searchable, we standardize on these set of characters:

```
  - _data/projects/ros-speech-command-node.yml:
    - Tag 'pid_control' contains invalid characters. Allowed characters: a-z, 0-9, +, #, . or -
    - Tag 'ros_noetic' contains invalid characters. Allowed characters: a-z, 0-9, +, #, . or -
    - Tag 'deep_learning' contains invalid characters. Allowed characters: a-z, 0-9, +, #, . or -
```

**Comment by shiftkey** [comment]
Closing this out due to inactivity. Please open a fresh pull request if you would still like it to be listed...

**Pull request closed without merging**